.npmignore

```diff
@@ -0,0 +1,6 @@
+**/.*
```

Comment:
Could we use the [`files`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#files) field in `package.json` instead of `.npmignore`, as there can be [unintended side-effects](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d).